### ISSUE TITLE
aria-hidden handled incorrectly

### ISSUE THREAD

**Issue by mfairchild365** [nvaccess/nvda#5833]
Take the following html code as an example

```
<a href="#"><span aria-hidden="true">this is content</span><span class="visually-hidden">This should be visually hidden</span></a>
```

NVDA only announces 'this is content'.

In this example, the class `.visually-hidden` is defined as:

```
.visually-hidden {
    font-size: 0;
    width: 1px;
    height: 1px;
    display: inline-block;
    overflow: hidden;
    position: absolute!important;
    border: 0!important;
    padding: 0!important;
    margin: 0!important;
    clip: rect(1px,1px,1px,1px);
}
```

Note that the second span containing 'This should be visually hidden' is not spoken at all, even when the class `.visually-hidden` is removed.

I don't have access to other screen readers on windows, but chromevox and voice over work as expected, reading only 'this should be visually hidden'.

**Comment by jcsteh** [comment]
I tested this with both Firefox and IE and i hear "link This should be 
visually hidden" in both when using keystrokes in browse mode.

I assume you were reading this with the mouse? Note that this is not how 
the majority of screen reader users will read. In this case, I do see 
the issue you describe. That's #4322. What to read in this case is 
actually a bit tricky, since mouse tracking is inherently tied to the 
screen (and thus "visible" content).

**Comment by Adriani90** [comment]
@mfairchild365, @jcsteh I cannot reproduce this issue in Firefox 65.0.1 and Chrome 71. But I can reproduce the issue only in IE when routing the mouse with nvda+shift+m or when moving the mouse on the screen.
cc: @leonardder, @nishimotz

**Comment by jcsteh** [comment]
Firefox changed its aria-hidden implementation last year such that aria-hidden content is no longer exposed to screen readers at all (as per the spec). So, this makes sense and we can consider this fixed as far as Firefox is concerned.

**Comment by Adriani90** [comment]
I am closing this issue as abandoned, given the missing updates from the initial author for so long time, and given that support for Internet Explorer has actually ended in 2022. The IE mode in Edge for backwards compatibility will be available until at least 2029, but I don't think that accessibility improvements will occur in that regard.

In Firefox, Edge and Chrome everything works as expected.
